### PR TITLE
Change return type of inline helpers from RppStatus to void

### PR DESCRIPTION
**Pull request by r-abishek**
@rrawther This PR just changes the return type of inline functions from RppStatus RPP_SUCCESS to void as discussed.

1. Change return type of inline functions from RppStatus to void
2. Remove the RPP_SUCCESS return statement for all these inline void functions

### CODE REVIEW

File: src/include/cl/rpp_cl_common.hpp [hunk 1]
```diff
@@ -74,7 +74,5 @@ inline RppStatus generate_gaussian_kernel_gpu(Rpp32f stdDev, Rpp32f *kernel, Rpp
     {
         kernel[i] /= sum;
```
Comment (rrawther) [inline src/include/cl/rpp_cl_common.hpp:75]: this can be done only if sum is guaranteed to be nonzero

**Review by rrawther — COMMENTED** [review]
@paveltc  to run tests
@r-abishek check possibility of division by zero

**Comment by r-abishek** [comment]
@paveltc This PR doesn't add any functionality, and just changes some return types, so just verifying a successful build for OpenCL and HIP are relevant.
@rrawther The sum is dependent on a non-zero factor and exp(x). And exp(x) cannot be zero, so the sum won't be zero.

**Comment by paveltc** [comment]
@rrawther @r-abishek 
I have built this PR with both HIP and OpenCL backends. Both build fine.
I have also built MIVisionX afterwards, it also builds fine for both but I did not have the dependencies installed for OpenCL so parts of MIVisionX were left out. As far as I know, the parts that get left out don't interact with RPP.

**Review by rrawther — APPROVED** [review]

**Pull request merged**